lib/chef/cookbook_site_streaming_uploader.rb

```diff
@@ -145,6 +146,7 @@ def make_request(http_verb, to_url, user_id, secret_key_filename, params = {}, h
         if url.scheme == "https"
           http.use_ssl = true
           http.verify_mode = verify_mode
```

Comment:
`DefaultSSLPolicy` will set the verify mode. Removing this will also allow you to remove the `verify_mode` method down on line 173ish.